packages/end-to-end/cypress/support/commands.js

```diff
@@ -1,25 +1,246 @@
+  cy.waitForReact();
+  cy.get('[data-cy=addFarm-continue]').should('not.be.disabled').click();
+});
+
+Cypress.Commands.add('onboardCompleteQuestions', (role) => {
+  // cy.clock();
+  cy.url().should('include', '/role_selection');
+  // cy.tick();
+  cy.get('[data-cy=roleSelection-continue]').should('exist').and('be.disabled');
+  cy.waitForReact();
+  cy.get('[data-cy=roleSelection-role]').should('exist').check(role, { force: true });
+  cy.get('[data-cy=roleSelection-continue]').should('not.be.disabled').click();
+  // cy.clock().then((clock) => {
+  //   clock.restore();
+  // });
+
+  // Give Concent
+  cy.url().should('include', '/consent');
+  cy.get('[data-cy=consentPage-content]', { timeout: 180 * 1000 }).should('exist');
+  cy.get('[data-cy=consent-continue]').should('exist').and('be.disabled');
+  cy.get('[data-cy=checkbox-component]').should('exist').click();
+  cy.get('[data-cy=consent-continue]').should('not.be.disabled').click();
+
+  // Interested in Organic
+  cy.log('Interested in Organic');
+
+  cy.url().should('include', '/certification/interested_in_organic');
+  cy.get('[data-cy=interestedInOrganic-continue]', { timeout: 180 * 1000 })
+    .should('exist')
+    .and('be.disabled');
+  cy.get('[data-cy=interestedInOrganic-select]').should('exist');
+  cy.get('[type="radio"]').first().check();
+  cy.get('[data-cy=interestedInOrganic-continue]').should('not.be.disabled').click();
+
+  cy.url().should('include', '/certification/selection');
+  cy.get('[data-cy=certificationSelection-continue]').should('exist').and('be.disabled');
+  cy.get('[data-cy=certificationSelection-type]').should('exist');
+  cy.get('[type="radio"]').first().check();
+  cy.get('[data-cy=certificationSelection-continue]').should('not.be.disabled').click();
+
+  // Select certifier
+  cy.get('[data-cy=certifierSelection-proceed]').should('exist').and('be.disabled');
+  cy.get('[data-cy=certifierSelection-item]').should('exist').eq(1).click();
+  let certifier;
+  cy.get('[data-cy=certifierSelection-item]')
+    .eq(1)
+    .then(function ($elem) {
+      certifier = $elem.text();
+      let end = certifier.indexOf('(');
+      let result = certifier.substring(1, end);
+      //click the proceed button and ensure test is on the certification summary view and the certification selected is displayed
+      cy.get('[data-cy=certifierSelection-proceed]').should('not.be.disabled').click();
+      cy.url().should('include', '/certification/summary');
+      cy.contains(result).should('exist');
+    });
+
+  //certification summary
+  cy.get('[data-cy=certificationSummary-continue]').should('exist').and('not.be.disabled').click();
+
+  // Outro
+  cy.url().should('include', '/outro');
+  cy.get('[data-cy=outro-finish]').should('exist').and('not.be.disabled').click();
+});
+
+Cypress.Commands.add('createFirstLocation', (fieldString) => {
+  cy.get('[data-cy=home-farmButton]').should('exist').and('not.be.disabled').click({ force: true });
```

Comment:
We could store these selector strings somewhere to avoid having to update them in several different files when there's a UI change, e.g.

`const FARM_BUTTON_SELECTOR = '[data-cy=home-farmButton]'`